app/src/main/java/vn/mbm/phimp/me/leafpic/activities/LFMainActivity.java

```diff
@@ -95,17 +95,19 @@
     private SelectAlbumBottomSheet bottomSheetDialogFragment;
     private SwipeRefreshLayout swipeRefreshLayout;
     private boolean hidden = false, pickMode = false, editMode = false, albumsMode = true, firstLaunch = true;
+    /*
+    editMode-  When true, user can select items by clicking on them one by one
+     */
 
     private View.OnLongClickListener photosOnLongClickListener = new View.OnLongClickListener() {
         @Override
         public boolean onLongClick(View v) {
             Media m = (Media) v.findViewById(R.id.photo_path).getTag();
-            if (!editMode) {
-                // If it is the first long press
-                mediaAdapter.notifyItemChanged(getAlbum().toggleSelectPhoto(m));
+            if (!editMode) {                                                                           // If first long press, go into selection mode (editMode = 1)
```

Comment:
@Nirvan101 
Not really the best way to add comments.
Use the comment block to add all the relevant info for a variable at a single place.
http://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#tag